docs/fides/docs/installation/configuration.md

```diff
@@ -4,23 +4,53 @@ Fidesctl supports two methods of configuration. The first is via a toml file, an
 
 By default the fidesctl CLI doesn't require a config file and will instead leverage the default values. These are very likely to be wrong however so it is recommended to always configure your settings properly.
 
+
 ## Configuration file
 
-After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a fidesctl configuration file:
+After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a default fidesctl configuration file and documentation for each section:
 
-```toml title="fidesctl.toml"
-[api]
-database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-log_destination = ""
-log_level = 20
-log_serialization = ""
+=== "default fidesctl.toml"
 
-[cli]
-server_url = "http://localhost:8080"
+    ```toml
+    [api]
+    database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
+
+    # Logging
+    log_destination = ""
+    log_level = INFO
+    log_serialization = ""
+
+    [cli]
+    local_mode = False
+    server_url = "http://localhost:8080"
+
+    [user]
+    encryption_key = "test_encryption_key"
+    ```
+
+=== "API Section"
+
+    | Name | Type | Default | Description |
+    | :----: | :----: | :-------: | :-----------: |
+    | test_database_url | String | ""| If the `FIDESCTL_TEST_MODE` environment variable is set to `True`, the `test_database_url` is used instead of the `database_url` to avoid overwriting production data. |
+    | database_url | String | postgres:fidesctl@fidesctl-db:5432/fidesctl | The PostgreSQL database connection string for the fidesctl database. __NOTE__: Do not include the driver here, fidesctl will do this for you. |
+    | log_destination | String | "" | The output location for log files. Accepts any valid file path. If left unset, no log files are produced. |
+    | log_level | Enum (String) | INFO | The minimum log entry level to produce. Also accepts: `TRACE`, `DEBUG`, `WARNING`, `ERROR`, or `CRITICAL` (case insensitive). |
+    | log_serialization | Enum (String) | "" | The format with which to produce log entries. If left unset, produces log entries formatted using the internal custom formatter. Also accepts: `"JSON"` (case insensitive). |
+
+=== "CLI Section"
+
+    | Name | Type | Default | Description |
+    | :----: | :----: | :-------: | :-----------: |
+    | local_mode | Boolean | False | When to `True`, the CLI will never attempt to call out to a fidesctl webserver. |
+    | server_url | String | "" | The URL for the fidesctl webserver that the CLI should connect to. |
+
+=== "User Section"
+
+    | Name | Type | Default | Description |
+    | :----: | :----: | :-------: | :-----------: |
+    | encryption_key | String | "" | The key used to encrypt the user's data stored in the database. |
```

Comment:
I break my own rule about leaking implementation details, but in this case I think it's valuable information. Without specifying the cryptography algorithm, users might think they're required to provide an AES256-compatible bytestring, for example.
```suggestion
    | encryption_key | String | "" | An arbitrary string used to encrypt the user data stored in the database. Encryption is implemented using PGP. |
```